packages/@aws-cdk/cx-api/lib/directories.ts

```diff
@@ -0,0 +1,25 @@
+import * as fs from 'fs';
```

Comment:
Relocating this file from the CLI package (`aws-cdk`), so that it can be re-used in the CLI and the Framework. Let me know if there is a better location for this!